tools/systemd-boot/build.yml

```diff
@@ -0,0 +1,6 @@
+image: systemd-boot
+network: true
```

Comment:
Why do we need `network: true`? All of the `apk add` are local using `linuxkit/alpine`, and `ADD https://github.com/systemd/systemd.git#v258.1` does not require it either. Or is it the `RUN meson ...` that needs it?